addon/styles/_frost-text.scss

```diff
@@ -15,7 +15,6 @@ $padding: 0 19px 0 6px;
 .frost-text {
   display: flex;
   align-items: center;
-  width: 175px;
```

Comment:
It's debatable - in some cases you want a default, but it's really hard to pick a good default - 100%? 175px? 10em?  It really changes according to the consuming context.  In general I would expect text to primarily be used in bunsen forms, so bunsen should be responsible for the width.

So is there harm in providing a global default?  Probably not, as long as it doesn't restrict the override capabilities/make it hard to get what you want.

The consequence of this removal is going to end up with a default width (either px or 100% anyway), so it's probably better to keep it - thoughts?